src/test/java/de/berlin/lostberlin/service/BusinessServiceTest.java

```diff
@@ -29,30 +43,30 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
-/*
-Unfortunately, I get the same problem with @JpaAuditing annotation even after it's been externalized to ApplicationConfig file. So I had to comment it out for making tests.
-I added @ContextConfiguration because ApplicationContext could not be loaded without it.
- */
 
 @RunWith(SpringRunner.class)
-@ContextConfiguration(classes = ApplicationConfig.class, initializers = ConfigFileApplicationContextInitializer.class)
-@SpringBootTest
+@DataJpaTest
+@AutoConfigureTestDatabase(replace= AutoConfigureTestDatabase.Replace.NONE)
```

Comment:
`@DataJpaTest` is to use an in-memory test db for your service layer. It is not enough since we want to test against our real database. The second line with `@AutoConfigureTestDatabase` is to use our MySQL.